python/ray/train/torch/torch_trainer.py

```diff
@@ -45,32 +46,33 @@ def train_loop_per_worker(config: Dict):
     Inside the ``train_loop_per_worker`` function, you can use any of the
```

Comment:
Ideally there would also be an example for the above paragraph somewhere, we can feel free to do that in another PR.

Reply:
(You can discard this, I saw the usage is already shown in the example below -- maybe add `(see example below)`.